repos/fdbt-site/src/data/auroradb.ts

```diff
@@ -621,8 +621,9 @@ export const insertPassengerType = async (
     passengerType: PassengerType | CompanionInfo[],
     name: string,
     isGroup: boolean,
+    maxGroupSize?: string,
 ): Promise<void> => {
-    const contents = JSON.stringify(passengerType);
+    const contents = maxGroupSize ? JSON.stringify({ maxGroupSize, passengerType }) : JSON.stringify(passengerType);
```

Comment:
Would it be better to drive this from isGroup? 
Wonder if actually we should split the group stuff out now as you'd always want to provide maxGroupSize is isGroup is true?